R/derive_vars_query.R

```diff
@@ -108,10 +125,6 @@ derive_vars_query <- function(dataset, dataset_queries) {
 
   # queries restructured
   queries_wide <- dataset_queries %>%
-    mutate(
-      TERMCHAR = toupper(TERMCHAR),
-      PREFIX_NAM = paste0(PREFIX, "NAM")
-    ) %>%
```

Comment:
I think this should not be removed.

Reply:
my mistake, this must've been accidentally done at some point